index.html

```diff
@@ -57,25 +76,25 @@ <h5 class="display-h5">@National Museum of Korea, Art Center Nabi and More</h5>
         </div>
 
     </section>
-    <section class="program mt-4 bg-dark py-4 container-fluid">
+    <section class="program mt-4 bg-dark py-4 container-fluid" id="program">
         <h4 class="text-center text-white fw-bold text-capitalize mt-4">main program</h4>
         <div class="horizontal-line mb-4"></div>
         <div class="row d-flex justify-content-center mb-5">
-            <div class="col-2">
-                <div class="card">
-                    <div class="card-body">
+            <div class="col-12 col-md-4 col-lg-2">
+                <div class="card  mb-3">
+                    <div class="card-body ">
```

Comment:
Your main program section still needs some spacing, **positioning** and **font sizing** adjustments on the mobile view.
- [ ] Adjust the **spacing** of contents, as can be seen in the original design, so that _white_ text description on the right side occupies more than 50% of its parent container
- [ ] The middle orange text have **bolder fonts** and are **vertically centered**
- [ ] The contents are **horizontally centered** in the parent container, with equal **padding** on the extreme right and left sides

**Your main program (mobile view):**
![image](https://user-images.githubusercontent.com/12745474/133426552-436359c0-dd41-422d-b51a-d44b743963aa.png)


**Your originalprogram (mobile view):**
![image](https://user-images.githubusercontent.com/12745474/133428961-33db38b1-1518-4ca8-a23e-0cb4c9dc884e.png)

Reply:
Well done